contributors/design-proposals/raw-block-pv.md

```diff
@@ -0,0 +1,571 @@
+# Local Raw Block Consumption via Persistent Volume Source
```

Comment:
I think we don't need "via Persistent Volume Source" on the title since this spec will cover both static and dynamic provisioning, right?